viewer/packages/cad-geometry-loaders/src/sector/culling/OrderSectorsByVisibilityCoverage.test.ts

```diff
@@ -21,12 +22,12 @@ describe('OrderSectorsByVisibilityCoverage', () => {
     new THREE.Box3(new THREE.Vector3(-1, -1, -1), new THREE.Vector3(1, 1, 1))
   ]);
   const cadModel = createStubModel('model', singleSectorScene, identityMatrix);
-  const occludingGeometryProvider: OccludingGeometryProvider = {
-    renderOccludingGeometry: jest.fn()
-  };
+
+  let renderManager: EffectRenderManager;
 
   beforeEach(() => {
     renderer = new THREE.WebGLRenderer({ context: glContext });
+    renderManager = new EffectRenderManager(renderer, new THREE.Scene(), new CadMaterialManager(), {});
```

Comment:
Technically, the CadMaterialManager should be a mock here. It would limit the blast range if someone goes wrong in the CadMaterialManager.

Reply:
Made the EffectRenderManager a mock instead